src/v2/view-builder/views/AutoRedirectView.js

```diff
@@ -15,6 +15,12 @@ const Body = BaseForm.extend({
     const app = this.options.appState.get('app');
     const user = this.options.appState.get('user');
 
+    // OKTA-635926: add user gesture for ov enrollment on android
+    if (Util.isAndroidOVEnrollment()) {
+      titleString = loc('oie.success.text.signingIn.with.appName.android.ov.enrollment', 'login');
+      return titleString;
+    }
```

Comment:
Displaying the OV enrollment subtitle should also be independent of the interstitial value.